### PR TITLE
Remove omitted field from ImportRoute

### LINKED ISSUE #230: Remove ImportRoute.omitted
We are not using a PoW consensus engine. Since the `omitted` field in `ImportRoute` is reserved only for PoW engines, it should be removed. (https://www.notion.so/codechain/Enacted-VS-Imported-VS-Omitted-ea5d04ae319d4da38f9c91aa940d49c5)

If we remove the `omitted` field, we don't need the `ImportRoute` struct anymore. Maybe we can change the return type of `BlockChain::insert_block` from `ImportRoute` to `bool`. In this case, `true` will indicate that the block is successfully inserted to the chain, and false will indicate that the block is already inserted.

### PR DESCRIPTION
**Pull request by Byeongjee**
Resolves #230 

We don't use the `omitted` field anymore.

I'm not sure if I should remove `ImportRoute` struct at all, or leave it. 
Please give a comment.

### CODE REVIEW

File: test/package.json [hunk 1]
```diff
@@ -25,28 +25,28 @@
...
     "prettier": "^1.19.1",
-    "ts-node": "^8.5.2",
     "ts-jest": "^23.1.3",
+    "ts-node": "^8.5.2",
     "tslint": "^5.20.1",
     "tslint-config-prettier": "^1.18.0",
-    "typedoc": "^0.14.2",
     "tslint-no-unused-expression-chai": "^0.1.4",
+    "typedoc": "^0.16.11",
     "typescript": "^3.7.2"
```
Comment (majecty) [inline test/package.json]: Please remove unrelated changes. Or you can create a commit for this.

Comment (Byeongjee) [reply test/package.json]: Sorry for making mistakes often.  This was not intended. I'll remove it.

**Comment by Byeongjee** [comment]
A problem of `ImportResult` is that it's already in use. https://github.com/CodeChain-io/foundry/blob/d8d4e7c0d18ff9ddf0418452b6a34c8ebafab1cb/core/src/client/mod.rs#L260

**Comment by majecty** [comment]
@Byeongjee How about `EnactedResult`?

**Comment by Byeongjee** [comment]
I was thinking about `InsertResult` or `ChainUpdateResult`.
`EnactResult` would be fine as well.

**Comment by Byeongjee** [comment]
Btw I'll move this struct into `blockchain` module.

**Comment by majecty** [comment]
I like `ChainUpdateResult`!

**Comment by Byeongjee** [comment]
I'm not sure where to relocate the `ChainUpdateResult`. Do you have a suggestion?

**Comment by majecty** [comment]
Using blockchain/mod.rs or Creating a new file in the src/blockchain/ directory seems good.

**Comment by Byeongjee** [comment]
I created `update_result` submodule.

File: core/src/blockchain/update_result.rs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use super::block_info::{BestBlockChanged, BestHeaderChanged};
+use ctypes::BlockHash;
+
+#[derive(Debug, PartialEq)]
+pub struct ChainUpdateResult {
+    /// Blocks that were validated by new block.
```
Comment (majecty) [inline core/src/blockchain/update_result.rs]: Please update the comment.
